public/js/src/appstore.tag

```diff
@@ -666,7 +688,8 @@ function AppStore(host, services) {
             error: function (xhr, status, err) {
                 var error = JSON.parse(xhr.responseText);
                 d('BridgeStore::bridge_lb_status::error', error);
-                RiotControl.trigger('flash_message', 'error', error.message, 1000 * 15);
+                // do not alert here, it's annoying
+                //RiotControl.trigger('flash_message', 'error', error.message, 1000 * 15);
```

Comment:
removed alert for if we get an error when getting lb status. This is very annoying IMO. Not relevant to this PR, but I think it's needed.